src/components/TextInputFocusable/index.js

```diff
@@ -198,6 +199,18 @@ class TextInputFocusable extends React.Component {
         });
     }
 
+    /**
+     * Move cursor to the end.
```

Comment:
Is this all that does? In that case can we name the method `moveCursorToEnd()` or something else more explicit than `updateSelection` ?

Reply:
Sure, changed the function name and updated the PR 👍